frc1388-2022/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -543,4 +550,30 @@ public void simulationInit() {
   public void setNeutralMode(NeutralMode mode) {
     m_driveTrainSubsystem.setNeutralMode(mode);
   }
+
+  public boolean isGuestJoysticksMoved(){
+    // boolean isGuestJoysticksMoved = (m_driveController.getLeftX()!= 0) || (m_driveController.getLeftY()!= 0) || (m_driveController.getRightX()!= 0) || (m_driveController.getRightY()!= 0);
+    boolean isGuestJoysticksMoved = (!isClosetoZero(m_driveController.getLeftX())) || (!isClosetoZero(m_driveController.getLeftY())) || (!isClosetoZero(m_driveController.getRightX())) || (!isClosetoZero(m_driveController.getRightY())) ;
+    return isGuestJoysticksMoved; 
+  }
+
+  public boolean isClosetoZero(double number){
+    boolean isClosetoZero = (number < DriveTrainConstants.DEADBAND && number > -DriveTrainConstants.DEADBAND);
+    return isClosetoZero;
+  }
+
+
+static class GuestModeEnabled{
+  private static boolean isGuestModeEnabled = true;
```

Comment:
`isGuestModeEnabled` is a class member, and should be named `m_isGuestModeEnabled` for consistency.